### PR TITLE
compactor: adjust interval for period <1-hour

### PR DESCRIPTION
**Pull request by yudai**
For #9474

cc/@gyuho, @fanminshi

### CODE REVIEW

File: compactor/periodic.go [hunk 1]
```diff
@@ -61,75 +61,129 @@ func newPeriodic(clock clockwork.Clock, h time.Duration, rg RevGetter, c Compact
...
+const retryDivisor = 10
+
+func (t *Periodic) getRetryInterval() time.Duration {
+	itv := t.period / retryDivisor
+	// we don't want to too aggressive retries
+	// and also jump between 6-minute through 60-minute
+	if itv < (6 * time.Minute) { // t.period is less than hour
+		// if t.period is less than 6-minute,
+		// retry interval is t.period.
+		// if we divide byretryDivisor, it's too aggressive
```
Comment (gyuho) [inline compactor/periodic.go]: `s/byretryDivisor/by retryDivisor/`?

**Comment by gyuho** [comment]
@yudai Me and @fanminshi have simpler solution. Will send another PR shortly for review. Thanks!

**Comment by yudai** [comment]
@gyuho @fanminshi, sure thing! I'm totally fine with that. Let's choose the simpler way!

**Comment by yudai** [comment]
Moving to #9485.

**Pull request closed without merging**